From 10d32cdb644c3c4bf44bc8fa9e8733f9565543f8 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 28 Nov 2016 14:02:23 +0800
Subject: [PATCH 1294/5242] MLK-13522 ARM: dts: enable tempmon node in dts

commit  c7ba00302794d7de61518def45a0282d73464ade from
https://source.codeaurora.org/external/imx/linux-imx.git

In commit: b1a04e6ed63,(MLK-13413), the tempmon node is
wrongly disabled. so fix it in this patch.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll.dtsi |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index d7916eb..9df9ba77 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -528,7 +528,6 @@
 				fsl,tempmon = <&anatop>;
 				fsl,tempmon-data = <&ocotp>;
 				clocks = <&clks IMX6SLL_CLK_PLL3_USB_OTG>;
-				status = "disabled";
 			};
 
 			usbphy1: usbphy@020c9000 {
-- 
1.7.9.5

